Tests/js/views/assets/ez-dashboardblockbasedview-tests.js

```diff
@@ -0,0 +1,222 @@
+            Y.Assert.areSame(1, blocks.length, 'The `addBlock` method should return an array of blocks containing only 1 block');
+        },
+
+        'Should render 2 blocks in the dashboard with correct order': function () {
+            var view = this.view.render(),
+                block1Config = {
+                    priority: 2,
+                    identifier: 'A'
+                },
+                block2Config = {
+                    priority: 1,
+                    identifier: 'B'
+                },
+                block1 = new Y.eZ.DashboardBlockBaseView(block1Config),
+                block2 = new Y.eZ.DashboardBlockBaseView(block2Config),
+                blocks;
+
+            view.set('active', true);
+            blocks = view.addBlock(block1);
+            blocks = view.addBlock(block2);
+
+            Y.Assert.areSame(view, block1.getTargets()[0], 'The dashboard view should be a target of a first block');
+            Y.Assert.areSame(view, block2.getTargets()[0], 'The dashboard view should be a target of a second block');
+            Y.Assert.isTrue(block1.get('active'), 'A first block should be active');
+            Y.Assert.isTrue(block2.get('active'), 'A second block should be active');
+            Y.Assert.areSame(2, view.get('container').one(SELECTOR_CONTENT).get('children').size(), 'Should render one block inside');
+            Y.Assert.isArray(blocks, 'The `addBlock` method should return an array of blocks');
+            Y.Assert.areSame(2, blocks.length, 'The `addBlock` method should return an array of blocks containing 2 blocks');
+
+            // PhantomJS 1.9.8 swap params in `sort` function of Array.
+            // This is fixed in PhantomJS 2.0
+            // See: https://github.com/ariya/phantomjs/issues/11090
+            if (!Y.UA.phantomjs) {
+                Y.Assert.areSame(block1Config.identifier, blocks[1].get('identifier'), 'Should order blocks correctly');
+            }
+        },
+
+        'Should not render a block when providing incorrect view instance': function () {
+            var view = this.view.render(),
+                block1Config = {
+                    priority: 2,
+                    identifier: 'A'
+                },
+                block1 = new Y.View(block1Config),
+                blocks;
+
+            view.set('active', true);
+            blocks = view.addBlock(block1);
+
+            Y.Assert.areNotSame(view, block1.getTargets()[0], 'The dashboard view should be a target of a first block');
+            Y.Assert.areSame(0, view.get('container').one(SELECTOR_CONTENT).get('children').size(), 'Should render one block inside');
+            Y.Assert.isArray(blocks, 'The `addBlock` method should return an array of blocks');
+            Y.Assert.areSame(0, blocks.length, 'The `addBlock` method should return an array of blocks containing 2 blocks');
+        },
+    });
+
+    eventsTest = new Y.Test.Case({
```

Comment:
events ? which events ? here it seems you are testing that the `active` flag is correctly forwarded so it could be named `forwardActiveTest`.